rts/at/tx/src/test/java/org/jboss/jbossts/star/test/SpecTest.java

```diff
@@ -45,7 +45,7 @@
  */
 public class SpecTest extends BaseTest {
     // jax-rs does not support TRACE, CONNECT and PATCH
-    private static enum HTTP_METHOD {HEAD, GET, POST, PUT, DELETE, OPTIONS}
+    private static enum HTTP_METHOD { HEAD, GET, POST, PUT, DELETE, OPTIONS }
```

Comment:
Is there a rule that will avoid this (since `{HEAD, GET, POST, PUT, DELETE, OPTIONS}` would be normal).

Reply:
I see, I agree that this would not be needed to be enforced but I can't find a way to do so. I can configure to ignore the curly brace but then it's ignored completely.
I think it's a restriction we need to live with for now. Or maybe I can define the enum records on a new line in this case?

Reply:
No that's fine, thanks for investigating though.